tests/parsec-cli-tests.sh

```diff
@@ -197,13 +198,18 @@ test_csr() {
 
         echo
         echo "- Creating a certificate signing request (CSR) from the test key."
-        run_cmd $PARSEC_TOOL_CMD create-csr --cn ${TEST_CN} --san ${TEST_SAN} --key-name $KEY >${MY_TMP}/${KEY}.csr
+        run_cmd $PARSEC_TOOL_CMD create-csr --cn ${TEST_CN} --san ${TEST_SAN} --serialNumber ${TEST_SERIAL} --key-name $KEY >${MY_TMP}/${KEY}.csr
         debug cat ${MY_TMP}/${KEY}.csr
 
         echo
         echo "- Using openssl to inspect the CSR content and verify the public key."
-        debug export TEXT_PARAM='-text'
```

Comment:
Why have you removed this option? For all commands we output text details in debug only to reduce output in normal mode.